### PR TITLE
Boxes are now correctly aligned (Community Newsletter) -> on page: meshery.io/programs

### LINKED ISSUE #886: Boxes are not correctly aligned and are small in size -> on page: meshery.io/programs
#### Description

Boxes are not correctly aligned and are small in size -> on page: meshery.io/programs
image

#### Expected Behavior

image

#### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by UtkarshMishra12**
Signed-off-by: Utkarsh Mishra <nasautkarsh@gmail.com>

**Description**
Boxes are now correctly aligned (Community Newsletter) -> on page: meshery.io/programs

This PR fixes #886 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by UtkarshMishra12** [comment]
@legendarykamal Can you please review this? 😊

File: css/program.css [hunk 1]
```diff
@@ -354,9 +356,9 @@ img {
     padding: 4.5rem 6rem !important;
   }
 
-  .programlist-subscribe .form-container .subscribe-inputbox{
+  /* .programlist-subscribe .form-container .subscribe-inputbox{
```
Comment (legendarykamal) [inline css/program.css:359]: if this code isn't needed then we can clean this from here instead of commenting,

**Review by legendarykamal — APPROVED** [review]
LGTM 🚀 

Although I found another issue but thats not related to this PR so I will open a fresh issue for it .

File: css/program.css (hunk 1 above)
Comment (UtkarshMishra12) [reply css/program.css:359]: Sure, I'll open a new PR for the new issue.

Comment (legendarykamal) [reply css/program.css:359]: no not this one , it was about this https://github.com/meshery/meshery.io/issues/898 .

You can remove the commented code in this PR itself despite commenting out was my say .

**Review by legendarykamal — DISMISSED** [review]
looks good :)

File: css/program.css [hunk 2]
```diff
@@ -354,10 +356,6 @@ img {
     padding: 4.5rem 6rem !important;
   }
 
-  .programlist-subscribe .form-container .subscribe-inputbox{
-    flex-direction: column;
```
Comment (leecalcote) [inline css/program.css:358]: Moving away from flex is not ideal. Can you use justify-content, align-self, align-items, or other while staying with flex? Static assignment of 2rem of padding is not as preferable as staying with flex. You can use `space-around` to help with padding while using flex. You can assign padding while using flex, too. But... try to keep using flex. 😉

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: css/program.css (hunk 2 above)
Comment (Chadha93) [reply css/program.css:358]: // @UtkarshMishra12 please update.

**Comment by UtkarshMishra12** [comment]
@leecalcote @Chadha93 You can check it now. 

Before

https://user-images.githubusercontent.com/67385503/202842676-3e1f5ace-5af0-4342-a4b7-65647d219934.mp4

Now

https://user-images.githubusercontent.com/67385503/202842683-5a4385d2-5863-46b9-8f40-b7b2d4a045e8.mp4

**Comment by UtkarshMishra12** [comment]
@Chadha93 Gentle Reminder 😄

File: css/subscribe.css [hunk 3]
```diff
@@ -15,6 +15,7 @@
   box-sizing: border-box;
   border-radius: 5px;
   padding: 15px;
+  flex: 0 0 calc(50% - 1rem);
```
Comment (leecalcote) [inline css/subscribe.css:18]: Ooooh! Fancy, @UtkarshMishra12. 😄 👏

**Review by leecalcote — APPROVED** [review]

**Pull request merged**